modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsSLA.testcase

```diff
@@ -1002,6 +1004,10 @@ definition {
 
 		var getInstanceId = selenium.getText("WorkflowAllItems#WORKFLOW_INSTANCE_ID");
 
+		while ((IsElementNotPresent(key_status = "check", locator1 = "WorkflowAllItems#DETAIL_SLA_STATUS")) && (maxIterations = "50")) {
+			Refresh();
+		}
+
```

Comment:
This is step is to ensure that the poshi will wait for the status update (after SLADefinitionsJobInterval) and don't  try to click before this